src/app/admin/add-play/add-play.component.html

```diff
@@ -0,0 +1,79 @@
+<div class="modal-header container">
+    <h4 class="modal-title" id="modal-basic-title">Add new play</h4>
+    <button type="button" class="close" aria-label="Close" (click)="closeModal('close')">
+        <span aria-hidden="true">&times;</span>
+    </button>
+</div>
+<div class="modal-body">
+    <form>
+        <div class="form-group">
```

Comment:
maybe a `formGroup` would be more easier to work with, instead of multiple `ngModel`